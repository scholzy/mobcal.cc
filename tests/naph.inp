 12.0   -1.22922420114188     -1.40420721744858      0.00042610203981 -0.037191
 12.0   -2.45330401367585     -0.69130073793194      0.00034999527952 -0.025029
 12.0   -2.45330167337039      0.69131202679023      0.00003408694334 -0.028537
 12.0   -1.22921949717604      1.40421267485682     -0.00021119415199 -0.028935
 12.0   0.00000706040396      0.71446783292695     -0.00013172634118   0.064432
 12.0   1.22923498722065      1.40420689840044     -0.00036270015405  -0.029001
 12.0   2.45331520666813      0.69130089839767     -0.00027367662383  -0.030688
 12.0   2.45331290800531     -0.69131177117277      0.00003770888153  -0.018830
 12.0   1.22923097301059     -1.40421298283324      0.00026857197061  -0.042131
 12.0   0.00000410731814     -0.71446843975848      0.00019053394466   0.079092
  1.0   -1.23644766465187     -2.49445412996690      0.00066645517929  0.144036
  1.0   -3.39344861227993     -1.24018433328003      0.00054348042195  0.131141
  1.0   -3.39344406790553      1.24019905485856     -0.00002688209329  0.131370
  1.0   -1.23643764447996      2.49445969117462     -0.00046286295800  0.141872
  1.0   1.23645769140218      2.49445393349853     -0.00061208855742   0.142429
  1.0   3.39345943969036      1.24018506123226     -0.00045259911868   0.131561
  1.0   3.39345546468005     -1.24019857775130      0.00010706970499   0.128990
  1.0   1.23644953628207     -2.49445988199284      0.00050972563274   0.145420
