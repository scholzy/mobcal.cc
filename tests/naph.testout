COM coords: 5.58146e-06	-7.02002e-08	3.28499e-05
12	-1.22923e-10	-1.40421e-10	3.93252e-14
12	-2.45331e-10	-6.91301e-11	3.17145e-14
12	-2.45331e-10	6.91312e-11	1.23704e-16
12	-1.22923e-10	1.40421e-10	-2.44044e-14
12	1.47895e-16	7.14468e-11	-1.64576e-14
12	1.22923e-10	1.40421e-10	-3.9555e-14
12	2.45331e-10	6.91301e-11	-3.06527e-14
12	2.45331e-10	-6.91312e-11	4.85898e-16
12	1.22923e-10	-1.40421e-10	2.35722e-14
12	-1.47414e-16	-7.14468e-11	1.57684e-14
1	-1.23645e-10	-2.49445e-10	6.33605e-14
1	-3.39345e-10	-1.24018e-10	5.10631e-14
1	-3.39345e-10	1.2402e-10	-5.9732e-15
1	-1.23644e-10	2.49446e-10	-4.95713e-14
1	1.23645e-10	2.49445e-10	-6.44938e-14
1	3.39345e-10	1.24019e-10	-4.85449e-14
1	3.39345e-10	-1.2402e-10	7.42198e-15
1	1.23644e-10	-2.49446e-10	4.76876e-14
Old coordinates:
-1.22923e-10	-1.40421e-10	3.93252e-14
-2.45331e-10	-6.91301e-11	3.17145e-14
-2.45331e-10	6.91312e-11	1.23704e-16
-1.22923e-10	1.40421e-10	-2.44044e-14
1.47895e-16	7.14468e-11	-1.64576e-14
1.22923e-10	1.40421e-10	-3.9555e-14
2.45331e-10	6.91301e-11	-3.06527e-14
2.45331e-10	-6.91312e-11	4.85898e-16
1.22923e-10	-1.40421e-10	2.35722e-14
-1.47414e-16	-7.14468e-11	1.57684e-14
-1.23645e-10	-2.49445e-10	6.33605e-14
-3.39345e-10	-1.24018e-10	5.10631e-14
-3.39345e-10	1.2402e-10	-5.9732e-15
-1.23644e-10	2.49446e-10	-4.95713e-14
1.23645e-10	2.49445e-10	-6.44938e-14
3.39345e-10	1.24019e-10	-4.85449e-14
3.39345e-10	-1.2402e-10	7.42198e-15
1.23644e-10	-2.49446e-10	4.76876e-14
New coordinates:
-6.7253e-11	1.74084e-10	-3.09217e-14
-2.06695e-10	1.49143e-10	-2.75774e-14
-2.54154e-10	1.9282e-11	-4.26087e-15
-1.63655e-10	-8.96946e-11	1.60009e-14
-2.45249e-11	-6.71057e-11	1.21819e-14
6.7253e-11	-1.74083e-10	3.11515e-14
2.06695e-10	-1.49143e-10	2.65156e-14
2.54154e-10	-1.92821e-11	3.65126e-15
1.63655e-10	8.96946e-11	-1.51687e-14
2.45249e-11	6.71057e-11	-1.14927e-14
-3.05074e-11	2.76732e-10	-4.84324e-14
-2.76156e-10	2.32968e-10	-4.36412e-14
-3.61298e-10	-6.5945e-17	-1.44878e-15
-2.01757e-10	-1.91847e-10	3.46432e-14
3.05073e-11	-2.76732e-10	4.95658e-14
2.76156e-10	-2.32968e-10	4.1123e-14
3.61298e-10	-8.84924e-26	3.79701e-26
2.01757e-10	1.91847e-10	-3.27595e-14
6.65598e-13
Y STARTING
Well depths along the x-, y-, and z-axes:
-0.00583	-0.00720415	-0.0129356

Well coordinates along the x-, y-, and z-axes:
6.017	5.1251	3.10168

Distance where e = 0 along the x-, y-, and z-axes:
5.69752	4.79896	2.68901

1.81504	0.00148929	468.485	0.171904	16.5874	0.000744662
2.25299	0.00210617	581.525	0.264869	44.5415	0.00254243
2.53387	0.00257952	654.022	0.335027	74.7167	0.00488535
2.75156	0.00297857	710.212	0.395067	106.248	0.0076644
2.93357	0.00333014	757.191	0.449061	138.662	0.0108188
3.09222	0.00364799	798.139	0.498945	171.657	0.0143078
3.23424	0.00394028	834.797	0.545829	205.026	0.0181022
3.36372	0.00421234	868.217	0.590407	238.608	0.0221782
3.4834	0.00446786	899.108	0.633168	272.291	0.0265185
3.59516	0.00470953	927.955	0.674449	305.971	0.0311069
3.70041	0.0049394	955.121	0.714516	339.576	0.0359314
3.80019	0.00515904	980.877	0.753571	373.04	0.0409812
3.8953	0.0053697	1005.43	0.791763	406.3	0.0462455
3.98639	0.0055724	1028.94	0.829226	439.31	0.0517164
4.07399	0.00576798	1051.55	0.866069	472.031	0.0573868
4.15851	0.00595714	1073.36	0.902379	504.42	0.0632493
4.24032	0.00614048	1094.48	0.938231	536.443	0.0692977
4.31972	0.0063185	1114.97	0.973698	568.076	0.0755276
4.39696	0.00649164	1134.91	1.00883	599.283	0.0819324
4.47226	0.00666029	1154.35	1.04368	630.045	0.0885087
4.54582	0.00682476	1173.33	1.07829	660.334	0.0952516
4.61779	0.00698537	1191.91	1.11271	690.124	0.102156
4.68834	0.00714236	1210.12	1.14697	719.405	0.109221
4.75758	0.00729598	1227.99	1.18109	748.145	0.116439
4.82564	0.00744643	1245.56	1.21513	776.339	0.123811
4.89262	0.0075939	1262.85	1.2491	803.956	0.13133
4.95863	0.00773855	1279.88	1.28303	830.991	0.138997
5.02373	0.00788056	1296.69	1.31694	857.417	0.146806
5.08803	0.00802005	1313.28	1.35087	883.231	0.154757
5.15158	0.00815715	1329.69	1.38482	908.41	0.162846
5.21445	0.00829199	1345.91	1.41883	932.938	0.17107
5.2767	0.00842467	1361.98	1.45291	956.802	0.179428
5.33841	0.00855529	1377.91	1.48709	979.998	0.187919
5.39961	0.00868395	1393.71	1.52138	1002.5	0.196539
5.46035	0.00881073	1409.38	1.5558	1024.3	0.205285
5.5207	0.00893571	1424.96	1.59038	1045.39	0.214158
5.58069	0.00905897	1440.44	1.62513	1065.74	0.223156
5.64036	0.00918057	1455.85	1.66007	1085.36	0.232276
5.69977	0.00930058	1471.18	1.69522	1104.23	0.241516
5.75894	0.00941907	1486.45	1.73061	1122.33	0.250877
5.81791	0.00953608	1501.67	1.76623	1139.65	0.260354
5.87673	0.00965167	1516.86	1.80213	1156.18	0.269948
5.93544	0.0097659	1532.01	1.83831	1171.92	0.279658
5.99405	0.0098788	1547.14	1.87479	1186.83	0.28948
6.05261	0.00999043	1562.25	1.91161	1200.92	0.299414
6.11116	0.0101008	1577.37	1.94877	1214.17	0.309459
6.16972	0.01021	1592.48	1.98629	1226.56	0.319614
6.22834	0.0103181	1607.61	2.02422	1238.1	0.329879
6.28705	0.010425	1622.77	2.06256	1248.75	0.340251
6.34587	0.0105308	1637.95	2.10134	1258.51	0.350729
6.40484	0.0106356	1653.17	2.14057	1267.37	0.361311
6.464	0.0107394	1668.44	2.1803	1275.31	0.371999
6.52339	0.0108422	1683.77	2.22055	1282.32	0.38279
6.58303	0.010944	1699.16	2.26133	1288.38	0.393682
6.64297	0.0110448	1714.63	2.3027	1293.48	0.404677
6.70323	0.0111448	1730.19	2.34467	1297.61	0.415771
6.76388	0.0112439	1745.84	2.38729	1300.75	0.426967
6.82492	0.0113421	1761.6	2.43057	1302.88	0.438258
6.88644	0.0114394	1777.48	2.47459	1303.99	0.449651
6.94843	0.011536	1793.48	2.51934	1304.06	0.461137
7.01097	0.0116317	1809.62	2.56489	1303.08	0.472721
7.0741	0.0117266	1825.91	2.61129	1301.03	0.484401
7.13786	0.0118208	1842.37	2.65857	1297.88	0.496174
7.20231	0.0119143	1859.01	2.70681	1293.63	0.508041
7.26752	0.012007	1875.84	2.75604	1288.24	0.520002
7.33354	0.012099	1892.88	2.80634	1281.71	0.532056
7.40043	0.0121903	1910.14	2.85776	1274.01	0.544201
7.46825	0.012281	1927.65	2.91038	1265.11	0.556435
7.53709	0.0123709	1945.42	2.96429	1255	0.568761
7.60704	0.0124603	1963.47	3.01956	1243.65	0.581178
7.67815	0.0125489	1981.83	3.07628	1231.04	0.593681
7.75055	0.012637	2000.51	3.13457	1217.13	0.606275
7.82433	0.0127245	2019.56	3.19454	1201.9	0.618955
7.89963	0.0128113	2038.99	3.25631	1185.32	0.631724
7.97653	0.0128976	2058.84	3.32002	1167.35	0.644579
8.0552	0.0129833	2079.15	3.38583	1147.98	0.657519
8.13578	0.0130684	2099.95	3.45392	1127.15	0.670545
8.21846	0.013153	2121.29	3.52447	1104.83	0.683655
8.30343	0.0132371	2143.22	3.59772	1080.97	0.69685
8.39091	0.0133206	2165.8	3.67393	1055.55	0.710128
8.48116	0.0134036	2189.09	3.75339	1028.49	0.723491
8.57447	0.0134861	2213.18	3.83643	999.759	0.736937
8.67115	0.013568	2238.13	3.92344	969.298	0.750462
8.77165	0.0136495	2264.07	4.0149	937.033	0.764072
8.87637	0.0137305	2291.1	4.11134	902.908	0.777761
8.9859	0.0138111	2319.37	4.21343	866.837	0.791532
9.10089	0.0138911	2349.05	4.32196	828.735	0.805384
9.22213	0.0139707	2380.35	4.43788	788.513	0.819314
9.35065	0.0140499	2413.52	4.56244	746.05	0.833324
9.4877	0.0141286	2448.89	4.69715	701.224	0.847413
9.63489	0.0142069	2486.89	4.84403	653.882	0.861582
9.79435	0.0142847	2528.04	5.0057	603.854	0.875827
9.96901	0.0143621	2573.13	5.18582	550.921	0.890151
10.1629	0.0144392	2623.18	5.38955	494.821	0.904551
10.3822	0.0145158	2679.78	5.62463	435.217	0.919029
10.6364	0.014592	2745.4	5.90345	371.661	0.933582
10.9422	0.0146678	2824.31	6.24768	303.516	0.948212
11.332	0.0147432	2924.94	6.70085	229.827	0.962918
11.888	0.0148182	3068.44	7.37448	148.885	0.977698
12.9696	0.0148929	3347.62	8.77745	56.5772	0.992554
Done with gst!
1.81504	20.8	13.8782
2.25299	17.4	12.6934
2.53387	15.8	12.0957
2.75156	14.8	11.7067
2.93357	14.1	11.4265
3.09222	13.6	11.222
3.23424	13.1	11.0138
3.36372	12.7	10.8444
3.4834	12.4	10.7155
3.59516	12.1	10.5851
3.70041	11.9	10.4972
3.80019	11.6	10.3641
3.8953	11.4	10.2744
3.98639	11.3	10.2292
4.07399	11.1	10.1383
4.15851	10.9	10.0465
4.24032	10.8	10.0003
4.31972	10.6	9.90729
4.39696	10.5	9.86045
4.47226	10.4	9.81338
4.54582	10.3	9.76609
4.61779	10.1	9.67081
4.68834	10	9.62281
4.75758	9.9	9.57458
4.82564	9.8	9.5261
4.89262	9.7	9.47737
4.95863	9.7	9.47737
5.02373	9.6	9.42839
5.08803	9.5	9.37916
5.15158	9.4	9.32966
5.21445	9.3	9.2799
5.2767	9.3	9.2799
5.33841	9.2	9.22988
5.39961	9.1	9.17958
5.46035	9.1	9.17958
5.5207	9	9.129
5.58069	8.9	9.07814
5.64036	8.9	9.07814
5.69977	8.8	9.027
5.75894	8.7	8.97556
5.81791	8.7	8.97556
5.87673	8.6	8.92383
5.93544	8.6	8.92383
5.99405	8.5	8.87179
6.05261	8.5	8.87179
6.11116	8.4	8.81945
6.16972	8.4	8.81945
6.22834	8.3	8.7668
6.28705	8.3	8.7668
6.34587	8.2	8.71383
6.40484	8.2	8.71383
6.464	8.1	8.66053
6.52339	8.1	8.66053
6.58303	8	8.6069
6.64297	8	8.6069
6.70323	7.9	8.55294
6.76388	7.9	8.55294
6.82492	7.8	8.49864
6.88644	7.8	8.49864
6.94843	7.7	8.44398
7.01097	7.7	8.44398
7.0741	7.7	8.44398
7.13786	7.6	8.38897
7.20231	7.6	8.38897
7.26752	7.5	8.3336
7.33354	7.5	8.3336
7.40043	7.5	8.3336
7.46825	7.4	8.27785
7.53709	7.4	8.27785
7.60704	7.3	8.22173
7.67815	7.3	8.22173
7.75055	7.2	8.16523
7.82433	7.2	8.16523
7.89963	7.2	8.16523
7.97653	7.1	8.10832
8.0552	7.1	8.10832
8.13578	7	8.05102
8.21846	7	8.05102
8.30343	6.9	7.99331
8.39091	6.9	7.99331
8.48116	6.9	7.99331
8.57447	6.8	7.93517
8.67115	6.8	7.93517
8.77165	6.7	7.87661
8.87637	6.7	7.87661
8.9859	6.6	7.81761
9.10089	6.6	7.81761
9.22213	6.5	7.75816
9.35065	6.5	7.75816
9.4877	6.4	7.69825
9.63489	6.4	7.69825
9.79435	6.3	7.63787
9.96901	6.2	7.57701
10.1629	6.2	7.57701
10.3822	6.1	7.51566
10.6364	6	7.4538
10.9422	5.9	7.39142
11.332	5.8	7.32851
11.888	5.6	7.20105
12.9696	5.4	7.07129
Done with bst!
Cycle  0: 61.454919
Cycle  1: 60.328843
Cycle  2: 62.425029
Cycle  3: 57.922135
Cycle  4: 59.130350
Cycle  5: 58.128832
Cycle  6: 59.670162
Cycle  7: 62.397295
Cycle  8: 59.845691
Cycle  9: 61.865472
Cycle 10: 62.309864
Cycle 11: 63.031749
Cycle 12: 61.320279
Cycle 13: 63.320834
Cycle 14: 60.859395
Cycle 15: 62.641711
Cycle 16: 64.347718
Cycle 17: 62.658024
Cycle 18: 65.056880
Cycle 19: 59.203457
Cycle 20: 61.681972
Cycle 21: 60.434965
Cycle 22: 62.143617
Cycle 23: 58.141812
Cycle 24: 59.872786
Cycle 25: 62.498592
Cycle 26: 58.058956
Cycle 27: 61.768510
Cycle 28: 61.346294
Cycle 29: 61.806256
Cycle 30: 59.440497
Cycle 31: 63.029767
Cycle 32: 60.004881
Cycle 33: 62.203113
Cycle 34: 62.116084
Cycle 35: 59.036218
Cycle 36: 61.879019
Cycle 37: 62.678438
Cycle 38: 62.361788
Cycle 39: 63.313053
Average CCS: 61.293381
