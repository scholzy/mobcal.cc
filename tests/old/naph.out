 input file name = naph.mjf                      
 input file label = naphthalene                   
 number of coordinate sets =    1
 number of atoms =  18
 coordinates in angstroms
 correction factor for coordinates = 1.0000E+00
 using a calculated (non-uniform) charge distribution
 total charge = 1.0000E+00
 total absolute charge = 1.4807E+00
 mass of ion = 1.2816D+02

         initial coordinates         mass   charge         LJ parameters

 center of mass coordinates =  5.5815E-06,-7.0200E-08, 3.2850E-05
 -1.2292E-10 -1.4042E-10  3.9325E-14  12 -3.7191E-02  1.3400E-03  3.0430E+00
 -2.4533E-10 -6.9130E-11  3.1715E-14  12 -2.5029E-02  1.3400E-03  3.0430E+00
 -2.4533E-10  6.9131E-11  1.2370E-16  12 -2.8537E-02  1.3400E-03  3.0430E+00
 -1.2292E-10  1.4042E-10 -2.4404E-14  12 -2.8935E-02  1.3400E-03  3.0430E+00
  1.4789E-16  7.1447E-11 -1.6458E-14  12  6.4432E-02  1.3400E-03  3.0430E+00
  1.2292E-10  1.4042E-10 -3.9555E-14  12 -2.9001E-02  1.3400E-03  3.0430E+00
  2.4533E-10  6.9130E-11 -3.0653E-14  12 -3.0688E-02  1.3400E-03  3.0430E+00
  2.4533E-10 -6.9131E-11  4.8590E-16  12 -1.8830E-02  1.3400E-03  3.0430E+00
  1.2292E-10 -1.4042E-10  2.3572E-14  12 -4.2131E-02  1.3400E-03  3.0430E+00
 -1.4741E-16 -7.1447E-11  1.5768E-14  12  7.9092E-02  1.3400E-03  3.0430E+00
 -1.2365E-10 -2.4945E-10  6.3361E-14   1  1.4404E-01  6.5000E-04  2.3800E+00
 -3.3935E-10 -1.2402E-10  5.1063E-14   1  1.3114E-01  6.5000E-04  2.3800E+00
 -3.3934E-10  1.2402E-10 -5.9732E-15   1  1.3137E-01  6.5000E-04  2.3800E+00
 -1.2364E-10  2.4945E-10 -4.9571E-14   1  1.4187E-01  6.5000E-04  2.3800E+00
  1.2365E-10  2.4945E-10 -6.4494E-14   1  1.4243E-01  6.5000E-04  2.3800E+00
  3.3935E-10  1.2402E-10 -4.8545E-14   1  1.3156E-01  6.5000E-04  2.3800E+00
  3.3934E-10 -1.2402E-10  7.4220E-15   1  1.2899E-01  6.5000E-04  2.3800E+00
  1.2364E-10 -2.4945E-10  4.7688E-14   1  1.4542E-01  6.5000E-04  2.3800E+00


 Lennard-Jones scaling parameters: eo= 1.3400E-03 ro= 3.0430E+00
 dipole constant = 2.3642E-59
 mobility constant = 9.3953E-21
 temperature = 2.9800E+02

 using RANLUX

 RANLUX LUXURY LEVEL SET BY RLUXGO : 3     P= 223
 RANLUX INITIALIZED BY RLUXGO FROM SEEDS      487587           0           0



 structural asymmetry parameter =  1.4720


 mobility calculation by MOBIL4 (HS scattering)

 number of Monte Carlo trajectories = 250000
 maximum number of reflections allowed = 30

 average PA mobility = 8.4834E-04
 inverse average PA mobility = 1.1788E+03
 average PA cross section = 6.4155E+01

 maximum number of reflections encountered =  4

     order   cross section
       1      6.49777E-19
       2      6.57162E-19
       3      6.57235E-19
       4      6.57236E-19

 average EHS mobility = 8.2810E-04
 inverse average EHS mobility = 1.2076E+03
 average EHS cross section = 6.5724E+01

 mobility calculation by MOBIL2 (trajectory method)

 global trajectory parameters

 sw1 = 5.0000E-05       sw2 = 5.0000E-03
 dtsf1 = 5.0000E-01     dtsf2 = 1.0000E-01
 inwr =  1              ifail =  100

 maximum extent orientated along x axis


 coordinates rotated by ROTATE

 theta= 0.0000E+00 phi= 1.8000E+02 gamma= 3.3992E+02

    1     -6.7253E-11      1.7408E-10     -3.0922E-14      1.0000E+00
    2     -2.0669E-10      1.4914E-10     -2.7577E-14      1.0000E+00
    3     -2.5415E-10      1.9282E-11     -4.2609E-15      1.0000E+00
    4     -1.6366E-10     -8.9695E-11      1.6001E-14      1.0000E+00
    5     -2.4525E-11     -6.7106E-11      1.2182E-14      1.0000E+00
    6      6.7253E-11     -1.7408E-10      3.1152E-14      1.0000E+00
    7      2.0669E-10     -1.4914E-10      2.6516E-14      1.0000E+00
    8      2.5415E-10     -1.9282E-11      3.6513E-15      1.0000E+00
    9      1.6366E-10      8.9695E-11     -1.5169E-14      1.0000E+00
   10      2.4525E-11      6.7106E-11     -1.1493E-14      1.0000E+00
   11     -3.0507E-11      2.7673E-10     -4.8432E-14      1.0000E+00
   12     -2.7616E-10      2.3297E-10     -4.3641E-14      1.0000E+00
   13     -3.6130E-10     -6.5945E-17     -1.4488E-15      1.0000E+00
   14     -2.0176E-10     -1.9185E-10      3.4643E-14      1.0000E+00
   15      3.0507E-11     -2.7673E-10      4.9566E-14      1.0000E+00
   16      2.7616E-10     -2.3297E-10      4.1123E-14      1.0000E+00
   17      3.6130E-10     -8.8492E-26      3.7970E-26      1.0000E+00
   18      2.0176E-10      1.9185E-10     -3.2759E-14      1.0000E+00
 test e =    7603853.4073588280     
 along x axis emax =-5.8300E-03eV rmax = 6.0170E+00A r00 = 5.6975E+00A
 along y axis emax =-7.2042E-03eV rmax = 5.1251E+00A r00 = 4.7990E+00A
 along z axis emax =-1.2936E-02eV rmax = 3.1017E+00A r00 = 2.6890E+00A


 t*= 1.9164E+01


 set-up gst integration - integration over velocity

     pgst        wgst         v         ke/kt       gst^5*     frac of
                                                exp(gst^2/tst)   sum

  2.7803E+00  1.6216E-02  7.1763E+02  4.0337E-01  1.1099E+02  8.1082E-03
  3.5130E+00  2.2933E-02  9.0675E+02  6.4399E-01  2.8102E+02  2.7684E-02
  4.0098E+00  2.8088E-02  1.0350E+03  8.3901E-01  4.4798E+02  5.3194E-02
  4.4147E+00  3.2433E-02  1.1395E+03  1.0170E+00  6.0651E+02  8.3455E-02
  4.7703E+00  3.6261E-02  1.2313E+03  1.1874E+00  7.5344E+02  1.1780E-01
  5.0963E+00  3.9722E-02  1.3154E+03  1.3552E+00  8.8651E+02  1.5579E-01
  5.4036E+00  4.2905E-02  1.3947E+03  1.5236E+00  1.0039E+03  1.9711E-01
  5.6996E+00  4.5867E-02  1.4711E+03  1.6951E+00  1.1042E+03  2.4149E-01
  5.9897E+00  4.8649E-02  1.5460E+03  1.8721E+00  1.1858E+03  2.8875E-01
  6.2784E+00  5.1281E-02  1.6205E+03  2.0569E+00  1.2472E+03  3.3871E-01
  6.5697E+00  5.3784E-02  1.6957E+03  2.2522E+00  1.2871E+03  3.9125E-01
  6.8680E+00  5.6175E-02  1.7727E+03  2.4613E+00  1.3038E+03  4.4623E-01
  7.1779E+00  5.8469E-02  1.8527E+03  2.6885E+00  1.2954E+03  5.0355E-01
  7.5055E+00  6.0676E-02  1.9372E+03  2.9395E+00  1.2598E+03  5.6312E-01
  7.8590E+00  6.2806E-02  2.0285E+03  3.2229E+00  1.1944E+03  6.2486E-01
  8.2507E+00  6.4866E-02  2.1296E+03  3.5522E+00  1.0959E+03  6.8870E-01
  8.7011E+00  6.6862E-02  2.2458E+03  3.9506E+00  9.5975E+02  7.5456E-01
  9.2498E+00  6.8800E-02  2.3875E+03  4.4645E+00  7.7936E+02  8.2239E-01
  9.9945E+00  7.0686E-02  2.5797E+03  5.2124E+00  5.4337E+02  8.9214E-01
  1.1357E+01  7.2522E-02  2.9315E+03  6.7309E+00  2.2553E+02  9.6374E-01


 set up b2 integration - integration over impact parameter

 minimum value of (1-cosX) = 5.0000E-04


     gst           b2max/ro2         b/A

  2.7803E+00      1.4700E+01      1.1667E+01
  3.5130E+00      1.2300E+01      1.0672E+01
  4.0098E+00      1.1200E+01      1.0184E+01
  4.4147E+00      1.0500E+01      9.8604E+00
  4.7703E+00      9.9000E+00      9.5746E+00
  5.0963E+00      9.5000E+00      9.3792E+00
  5.4036E+00      9.1000E+00      9.1796E+00
  5.6996E+00      8.8000E+00      9.0270E+00
  5.9897E+00      8.5000E+00      8.8718E+00
  6.2784E+00      8.3000E+00      8.7668E+00
  6.5697E+00      8.0000E+00      8.6069E+00
  6.8680E+00      7.8000E+00      8.4986E+00
  7.1779E+00      7.6000E+00      8.3890E+00
  7.5055E+00      7.4000E+00      8.2779E+00
  7.8590E+00      7.2000E+00      8.1652E+00
  8.2507E+00      7.0000E+00      8.0510E+00
  8.7011E+00      6.8000E+00      7.9352E+00
  9.2498E+00      6.5000E+00      7.7582E+00
  9.9945E+00      6.2000E+00      7.5770E+00
  1.1357E+01      5.8000E+00      7.3285E+00


 number of complete cycles (itn) =    40
 number of velocity points (inp) =    20
 number of random points (imp) =    50
 total number of points =  40000


 summary of mobility calculations

 cycle     cs/A^2      avge cs/A^2        Ko^-1       avge Ko^-1
   1     6.1467E+01     6.1467E+01     1.1294E+03     1.1294E+03
   2     5.7651E+01     5.9559E+01     1.0593E+03     1.0943E+03
   3     6.4421E+01     6.1180E+01     1.1837E+03     1.1241E+03
   4     6.0660E+01     6.1050E+01     1.1146E+03     1.1217E+03
   5     6.4785E+01     6.1797E+01     1.1903E+03     1.1354E+03
   6     5.8893E+01     6.1313E+01     1.0821E+03     1.1265E+03
   7     6.3655E+01     6.1647E+01     1.1696E+03     1.1327E+03
   8     5.8825E+01     6.1295E+01     1.0808E+03     1.1262E+03
   9     5.9855E+01     6.1135E+01     1.0998E+03     1.1233E+03
  10     5.8883E+01     6.0909E+01     1.0819E+03     1.1191E+03
  11     6.2866E+01     6.1087E+01     1.1551E+03     1.1224E+03
  12     6.1335E+01     6.1108E+01     1.1270E+03     1.1228E+03
  13     5.9371E+01     6.0974E+01     1.0909E+03     1.1203E+03
  14     6.1502E+01     6.1012E+01     1.1300E+03     1.1210E+03
  15     5.8293E+01     6.0831E+01     1.0711E+03     1.1177E+03
  16     5.7966E+01     6.0652E+01     1.0651E+03     1.1144E+03
  17     5.8464E+01     6.0523E+01     1.0742E+03     1.1120E+03
  18     5.9029E+01     6.0440E+01     1.0846E+03     1.1105E+03
  19     6.8729E+01     6.0876E+01     1.2628E+03     1.1185E+03
  20     5.9008E+01     6.0783E+01     1.0842E+03     1.1168E+03
  21     5.7740E+01     6.0638E+01     1.0609E+03     1.1141E+03
  22     6.2032E+01     6.0701E+01     1.1398E+03     1.1153E+03
  23     6.3829E+01     6.0837E+01     1.1728E+03     1.1178E+03
  24     6.4220E+01     6.0978E+01     1.1800E+03     1.1204E+03
  25     6.9519E+01     6.1320E+01     1.2773E+03     1.1267E+03
  26     5.9906E+01     6.1266E+01     1.1007E+03     1.1257E+03
  27     6.0532E+01     6.1238E+01     1.1122E+03     1.1252E+03
  28     6.0510E+01     6.1212E+01     1.1118E+03     1.1247E+03
  29     6.0275E+01     6.1180E+01     1.1075E+03     1.1241E+03
  30     6.4577E+01     6.1293E+01     1.1865E+03     1.1262E+03
  31     6.0210E+01     6.1258E+01     1.1063E+03     1.1255E+03
  32     6.1731E+01     6.1273E+01     1.1342E+03     1.1258E+03
  33     5.8439E+01     6.1187E+01     1.0737E+03     1.1242E+03
  34     6.4256E+01     6.1277E+01     1.1806E+03     1.1259E+03
  35     6.4194E+01     6.1361E+01     1.1795E+03     1.1274E+03
  36     5.7865E+01     6.1264E+01     1.0632E+03     1.1256E+03
  37     6.0909E+01     6.1254E+01     1.1191E+03     1.1255E+03
  38     6.2238E+01     6.1280E+01     1.1435E+03     1.1259E+03
  39     5.7592E+01     6.1185E+01     1.0582E+03     1.1242E+03
  40     5.5448E+01     6.1042E+01     1.0188E+03     1.1216E+03


 average values for q1st

     gst2        wgst        q1st
  7.7302E+00  1.6216E-02  7.8862E+00
  1.2341E+01  2.2933E-02  5.7201E+00
  1.6079E+01  2.8088E-02  5.1924E+00
  1.9490E+01  3.2433E-02  4.9672E+00
  2.2756E+01  3.6261E-02  4.4780E+00
  2.5972E+01  3.9722E-02  4.4902E+00
  2.9199E+01  4.2905E-02  4.7571E+00
  3.2486E+01  4.5867E-02  4.3775E+00
  3.5877E+01  4.8649E-02  4.2376E+00
  3.9418E+01  5.1281E-02  4.1145E+00
  4.3161E+01  5.3784E-02  4.1752E+00
  4.7169E+01  5.6175E-02  3.9074E+00
  5.1522E+01  5.8469E-02  4.0865E+00
  5.6333E+01  6.0676E-02  3.7807E+00
  6.1765E+01  6.2806E-02  4.2690E+00
  6.8074E+01  6.4866E-02  3.9964E+00
  7.5709E+01  6.6862E-02  3.6457E+00
  8.5558E+01  6.8800E-02  4.0261E+00
  9.9891E+01  7.0686E-02  3.6218E+00
  1.2899E+02  7.2522E-02  3.4664E+00


 mean OMEGA*(1,1) = 2.0983E+00
 standard deviation = 1.0263E-01
 standard error of mean = 1.6227E-02


 f value for second order correction= 1.0000E+00
 (integrations for second order correction are not
 accurate, check if correction becomes significant)

 omega*12 = 1.9503E+00  omega*13 = 1.8113E+00  omega*22 = 2.1982E+00
       u2 = 1.5295E+01         w = 3.1230E-02     delta = 1.0625E-05


 average (second order) TM mobility = 8.9162E-04
 inverse average (second order) TM mobility = 1.1216E+03
 average TM cross section = 6.1042E+01



 SUMMARY

 program version = junkn.f
 input file name = naph.mjf                      
 input file label = naphthalene                   
 temperature = 2.9800E+02
 using RANLUX with seed integer =  487587
 structural asymmetry parameter = 1.4720E+00

 mobility calculation by MOBIL4 (HS scattering)

 number of Monte Carlo trajectories = 250000
 maximum number of reflections encountered =  4

 inverse average PA mobility = 1.1788E+03
 average PA cross section = 6.4155E+01

 inverse average EHS mobility = 1.2076E+03
 average EHS cross section = 6.5724E+01

 mobility calculation by MOBIL2 (trajectory method)

 trajectory parameters
 sw1 = 5.0000E-05       sw2 = 5.0000E-03
 dtsf1 = 5.0000E-01     dtsf2 = 1.0000E-01

 number of complete cycles (itn) =    40
 number of velocity points (inp) =    20
 number of random points (imp) =    50
 total number of points =  40000

 inverse average (second order) TM mobility = 1.1216E+03
 average TM cross section = 6.1042E+01
 standard deviation (percent) = 4.8909E+00
 number of failed trajectories =   0
